tools/docker/Dockerfile

```diff
@@ -5,27 +5,36 @@
 # Inspired by:
 # https://hub.docker.com/r/runmymind/docker-android-sdk/~/dockerfile/
 
-FROM ubuntu:17.10
+FROM ubuntu:18.04
 
-MAINTAINER Sebastian Kaspari "skaspari@mozilla.com"
+ENV ANDROID_BUILD_TOOLS "27.0.3"
+ENV ANDROID_SDK_VERSION "3859397"
```

Comment:
Where does this number come from? Perhaps add a comment where to get it.

Reply:
👍